lib/erlef_web/views/view_helpers.ex

```diff
@@ -98,6 +98,11 @@ defmodule ErlefWeb.ViewHelpers do
     !!assigns[:current_user]
   end
 
+  def allowed_to_post?(assigns) do
+    true
```

Comment:
Looks like `true` should be removed here.

Reply:
Great catch!